CMakeLists.txt

```diff
@@ -133,3 +133,11 @@ add_subdirectory(core)
 
 configure_file(cmake/Cabana.pc.cmakein Cabana.pc @ONLY)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/Cabana.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+
+find_package(CLANG_FORMAT 7.0.1)
+if(CLANG_FORMAT_FOUND)
+  file(GLOB_RECURSE FORMAT_SOURCES core/*.cpp core/*.hpp)
```

Comment:
I would have gone for something like `git ls-files | grep -E "\.hpp$|\.cpp$"` but that works.

Reply:
Yeah, I just wanted to avoid the usage of more external tools like git and grep.